crc/services/temp_migration_service.py

```diff
@@ -0,0 +1,262 @@
+# from crc import app, session
+from crc.models.file import FileModel, FileModelSchema, FileDataModel, CONTENT_TYPES
+from crc.models.workflow import WorkflowSpecModel, WorkflowSpecModelSchema, WorkflowSpecCategoryModel, WorkflowSpecCategoryModelSchema
+from crc.services.file_service import FileService
+from crc.services.spec_file_service import SpecFileService
+from crc.services.workflow_service import WorkflowService
+from sqlalchemy import desc
+
+import json
+import os
+
+
+# SYNC_FILE_ROOT = SpecFileService.get_sync_file_root()
+
+
+class FromFilesystemService(object):
+
+    @staticmethod
+    def process_directory(directory):
+        files = []
+        directories = []
+        directory_items = os.scandir(directory)
+        for item in directory_items:
+            if item.is_dir():
+                directories.append(item)
+            elif item.is_file():
+                files.append(item)
+
+        return files, directories
+
+    @staticmethod
+    def process_workflow_spec(json_file, directory):
+        file_path = os.path.join(directory, json_file)
+
+        with open(file_path, 'r') as f_open:
+            data = f_open.read()
+            data_obj = json.loads(data)
```

Comment:
It should be possible to load this json into a model with marshmallow, then you aren't hard-coding the strings coming and going from the file.  I see this is a "temp migration service" but I suspect some form of this will live on so we can update directly from the file system when needed.